example/tests/test_something.py

```diff
@@ -0,0 +1,9 @@
+def test_something():
```

Comment:
Nice name btw

Reply:
thanks?

umm is it a problem if almost all of the names in my tests look like SomeClass test_something test_something_else etc?)))

Reply:
:)
It's not very good, but it's not terrible though. Let's leave it as is.